app/models/concerns/smtp_config/providers_supportable.rb

```diff
@@ -44,14 +44,14 @@ module SmtpConfig::ProvidersSupportable
   #
   # @return [String]
   def address
-    host || PROVIDERS.dig(provider.to_sym, :address)
+    host.present? ? host : PROVIDERS.dig(provider.to_sym, :address)
```

Comment:
You could also do `host.presence ||` 😊

Reply:
Oh wow, TIL! Thanks for taking a look at these already-closed PRs, Kasper. 😄